File: kyw0716/level0/2차원으로 만들기.js
function solution(num_list, n) {
  const answer = [];
  let tempArray = [];

  for (let i = 0; i < num_list.length; i++) {
    if (tempArray.length === n) {
      answer.push(tempArray);
      tempArray = [];
    }
    tempArray.push(num_list[i]);
  }

  answer.push(tempArray);

  return answer;
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120842

File: sk000801/입문/2차원으로 만들기.java
import java.util.*;
class Solution {
    public int[][] solution(int[] num_list, int n) {
        int[][] answer = new int[num_list.length/n][n];
        for(int i = 0; i < num_list.length/n; i++) {
            answer[i] = Arrays.copyOfRange(num_list, n*i, n*(i+1));
        }
        return answer;
    }
}